app/model/fee-policy/helpers.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fee_policy
+
+func getInterfaceValue(input interface{}, key string) (interface{}, bool) {
+	mobj, ok := input.(map[interface{}]interface{})
+	if !ok {
+		return nil, false
+	}
+
+	for eleKey, eleValue := range mobj {
+		if eleKey.(string) == key {
+			return eleValue, true
+		}
+	}
+
+	return nil, false
+}
+
+func networkFound(networks []uint64, networkId uint64) bool {
+	if networks != nil && networkId > 0 {
```

Comment:
These checks aren't necessary. If `networks` is nil the for cycle won't start. If the `networkId == 0` then it just won't find it. If we want to leave the `networkId` check we should convert it to an early exit:
`if networkId == 0 { return false }`

Reply:
The method name is changed to `networkAllowed` be more clear. Actually the function is checking for restriction (true by default).